CHANGELOG.rst

```diff
@@ -14,6 +14,8 @@ Backward-incompatible changes:
 
 - Dropped support for Python 3.3.
   `#677 <https://github.com/pyca/pyopenssl/pull/677>`_
+- Removed the deprecated ``OpenSSL.rand`` module. ``os.urandom`` should be used instead.
```

Comment:
Please:

1. add a rationale why we choose to disregard our own policy (1 sentence is just fine)
2. make it `os.urandom()` – marking functions/methods with parentheses is nicer to read/understand.

thanks!